src/idl.ts

```diff
@@ -1332,89 +1332,83 @@ export type Uxd = {
     {
       name: 'collectProfitOfMercurialVaultDepository';
       accounts: [
-        {
-          name: 'authority';
-          isMut: false;
-          isSigner: true;
-          docs: ['#1'];
-        },
         {
           name: 'payer';
           isMut: true;
           isSigner: true;
-          docs: ['#2'];
+          docs: ['#1'];
         },
         {
           name: 'controller';
           isMut: true;
           isSigner: false;
-          docs: ['#3'];
+          docs: ['#2'];
         },
         {
           name: 'depository';
           isMut: true;
           isSigner: false;
-          docs: ['#4'];
+          docs: ['#3'];
         },
         {
           name: 'collateralMint';
           isMut: false;
           isSigner: false;
-          docs: ['#5'];
+          docs: ['#4'];
         },
         {
-          name: 'authorityCollateral';
+          name: 'profitsBeneficiaryCollateral';
           isMut: true;
           isSigner: false;
-          docs: ['#6'];
+          docs: ['#5'];
         },
         {
           name: 'depositoryLpTokenVault';
           isMut: true;
           isSigner: false;
           docs: [
-            '#7',
+            '#6',
             'Token account holding the LP tokens minted by depositing collateral on mercurial vault'
           ];
         },
         {
           name: 'mercurialVault';
           isMut: true;
           isSigner: false;
-          docs: ['#8'];
+          docs: ['#7'];
```

Comment:
I didnt know the docs were included on our version of anchor, I thought that didnt work, but if it does work its nice!

Reply:
no it didn't work, would hv deserialisating error with program build with older version anchor like us, thx for reminding